src/PrestaShopBundle/Install/Install.php

```diff
@@ -891,6 +891,7 @@ public function configureShop(array $data = [])
     public function getModulesList()
```

Comment:
Wah 😮  I did not know about this hardcoded list. It's a key item to be aware of though.